src/nplinker/scoring/rosetta/rosetta.py

```diff
@@ -536,12 +536,12 @@ def export_to_csv(self, filename):
 
             csvwriter.writerow([
                 'nplinker spectrum ID', 'spectrum ID', 'GNPS ID',
-                'spectral score', 'nplinker BGC ID', 'BGC ID', 'MiBIG BGC ID',
+                'spectral score', 'BGC ID', 'MiBIG BGC ID',
```

Comment:
Should the column names include spaces?

Reply:
it should be fine since it's csv file using comma as separator. I prefer not changing it at the moment.